modules/gapi/src/compiler/gcompiler.cpp

```diff
@@ -366,13 +368,17 @@ cv::gimpl::GCompiler::GPtr cv::gimpl::GCompiler::generateGraph()
     return makeGraph(m_c.priv().m_ins, m_c.priv().m_outs);
 }
 
-void cv::gimpl::GCompiler::runPasses(ade::Graph &g)
+void cv::gimpl::GCompiler:: runPasses(ade::Graph &g)
 {
     m_e.runPasses(g);
     GAPI_LOG_INFO(NULL, "All compiler passes are successful");
 }
 
-void cv::gimpl::GCompiler::compileIslands(ade::Graph &g)
+void cv::gimpl::GCompiler::compileIslands(ade::Graph &g) {
```

Comment:
Please follow the same coding style as in the rest of this file. `{` should be on a new line

Reply:
Fixed